docs/node/run-your-node/advanced/signing-key-manager-policy.md

```diff
@@ -0,0 +1,197 @@
+# Signing Key Manager Policy
+
+This guide will describe how to print and sign an Oasis key manager policy.
+
+:::info
+
+These instructions are only applicable if you are part of a key manager policy
+signer set.
+
+:::
+
+## Prerequisites
+
+### Oasis Node Binary
+
+Make sure you have followed the [Oasis Node binary installation guide] and have
+the Oasis Node binary installed on your system.
+
+### Entity
+
+Similarly to other things, an entity's private key is used to sign a key manager
+policy.
+
+The trusted key manager policy signer set is hard-coded in the key manager's
```

Comment:
I don't think the KM set implementation should go under the EXAMPLE admonition (we're using Example admonition frequently in runtime and dApp tutorials). Something more straight forward like this and then remove the EXAMPLE block below?

```suggestion
The trusted key manager policy signer set is hard-coded in the [key manager's source code][oasis-km-signer-set].
```

Reply:
> I don't think the KM set implementation should go under the EXAMPLE admonition (we're using Example admonition frequently in runtime and dApp tutorials).

Ok, agreed. Especially, if EXAMPLE admonitions are used differently in other parts of the docs.

I don't want the docs to imply that the Oasis Key Manager instance is the only one, so I tried to keep that distinction in the updated variant.